src/backend/distributed/operations/worker_node_manager.c

```diff
@@ -267,8 +267,11 @@ ErrorIfCoordinatorNotAddedAsWorkerNode()
 		return;
 	}
 
-	ereport(ERROR, (errmsg("could not find the coordinator node in "
-						   "metadata as it is not added as a worker")));
+	ereport(ERROR, (errmsg("operation is not allowed when coordinator "
+						   "is not added into metadata"),
+					errhint("Consider running \"SELECT citus_set_coordinator_host"
```

Comment:
maybe use consistent wording with others:

```
							errhint("Use SELECT citus_set_coordinator_host('<hostname>') "
									"to configure the coordinator hostname")));

```


feel free to add port param as well here as it might be useful